src/views/inspector/InspectorViewProvider.ts

```diff
@@ -94,15 +95,17 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
       this.featureConfigsMap = {}
 
       this.selectedKey = ''
+
       // Send variables data to the webview to initialize fuse
-      this.postMessageToWebview({
-        type: 'variables',
-        value: JSON.stringify(this.orderedVariables),
-      })
-      this.postMessageToWebview({
-        type: 'features',
-        value: JSON.stringify(this.orderedFeatures),
-      })
+      const variableOptions = this.orderedVariables.map(variable => ({
+        label: variable.key
+      }))
+      const featureOptions = this.orderedFeatures.map(feature => ({
+        label: feature.name || feature.key,
+        value: feature._id
+      }))
+      this.postMessageToWebview({type: 'searchData', searchType: 'variables', value: JSON.stringify(variableOptions)})
```

Comment:
it seems like we could benefit from using an enum for the `searchType` since it's passed as a param in a few places